daphne/src/taskprov.rs

```diff
@@ -0,0 +1,171 @@
+use crate::{
+    messages::{
+        taskprov::{QueryConfigVar, TaskConfig, VdafType, VdafTypeVar},
+        Extension, HpkeConfig, Id, ReportMetadata,
+    },
+    vdaf::VdafVerifyKey,
+    DapAbort, DapError, DapQueryConfig, DapTaskConfig, DapVersion, Prio3Config, VdafConfig,
+};
+use prio::codec::Decode;
+use ring::{
+    hkdf::{Prk, Salt, HKDF_SHA256},
+    hmac::HMAC_SHA256,
+    hmac::{sign, Key},
+};
+use std::str;
+use url::Url;
+
+/// SHA-256 of "dap-taskprov-00"
+#[allow(dead_code)]
+pub(crate) const TASK_PROV_SALT: [u8; 32] = [
+    0x4d, 0x63, 0x1a, 0xeb, 0xa8, 0xdf, 0xe0, 0x1b, 0x34, 0x4c, 0x29, 0x2d, 0x17, 0xba, 0x34, 0x9a,
+    0x78, 0x97, 0xbf, 0x64, 0x88, 0x00, 0x55, 0x1c, 0x0d, 0x75, 0x32, 0xab, 0x61, 0x4b, 0xe2, 0x21,
+];
+
+/// Compute the task id of a serialized task config.
+pub fn compute_task_id(serialized: &[u8]) -> Id {
```

Comment:
Can/should this be `pub(crate)`?

Reply:
No, because daphne_worker_test needs it.